lib/plugins/aws/package/compile/functions.js

```diff
@@ -25,6 +25,11 @@ const defaultCors = {
   ],
   allowedMethods: ['*'],
 };
+const runtimeManagementMap = new Map([
+  ['auto', 'Auto'],
+  ['onFunctionUpdate', 'Function update'],
```

Comment:
This doesn't match the Cloudformation docs:- Or am I missing something here?
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-runtimemanagementconfig.html#cfn-lambda-function-runtimemanagementconfig-updateruntimeon.

Reply:
Sorry for bringing things up, saw this in the release and as I was updating the schema, noticed this here.
It could be that CF is wrong here, but do let me know @ROSeaboyer and @medikoo I've submitted a [PR](https://github.com/serverless/serverless/pull/11778)

Reply:
Indeed it should be `FunctionUpdate` (cc @ROSeaboyer), PR with aa fix is welcome!

Reply:
Already created one:- [Here you go](https://github.com/serverless/serverless/pull/11778)